### PR TITLE
Reviewed nutrition facts for Authorization section in Guides

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Reviewed nutrition facts in Authorization section
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-456161](https://oktainc.atlassian.net/browse/OKTA-456161)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md [hunk 1]
```diff
@@ -6,24 +6,24 @@ layout: Guides
...
+* Customize Okta tokens with a dynamic allow list.
+* Use a dynamic allow list with an authorization server.
 
 **What you need**
 
-* An Okta Developer Edition organization. [Create an org for free](https://developer.okta.com/signup).
-* An [OpenID Connect client application](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-oidc) in Okta with at least [one user assigned to it](https://help.okta.com/okta_help.htm?id=ext-assign-apps).
-* A [group in Okta](https://help.okta.com/okta_help.htm?id=ext_Directory_Groups) with at least one person assigned to it.
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* [OpenID Connect client application](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-oidc) in Okta with at least [one user assigned to it](https://help.okta.com/okta_help.htm?id=ext-assign-apps).
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md:21]:
```suggestion
* [OpenID Connect client application](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-oidc) in Okta with at least [one user assigned to it](https://help.okta.com/okta_help.htm?id=ext-assign-apps)
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md [hunk 2]
```diff
@@ -6,24 +6,24 @@ layout: Guides
...
+* A [Group in Okta](https://help.okta.com/okta_help.htm?id=ext_Directory_Groups) with at least one person assigned to it.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md:22]:
```suggestion
* A [Group in Okta](https://help.okta.com/okta_help.htm?id=ext_Directory_Groups) with at least one person assigned to it
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/main/index.md [hunk 3]
```diff
@@ -3,22 +3,26 @@ title: Customize tokens returned from Okta with custom claims
...
-How to create your own custom claims for tokens.
+Create custom claims for tokens.
 
 **What you need**
 
-* An Okta developer org. [Create an org for free](/signup).
-* An [OpenID Connect client application](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-oidc) in Okta with at least [one user assigned to it](https://help.okta.com/okta_help.htm?id=ext-assign-apps).
-* A [group in Okta](https://help.okta.com/okta_help.htm?id=ext_Directory_Groups) with at least one person assigned to it.
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* [OpenID Connect client application](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-oidc) in Okta with at least [one user assigned to it](https://help.okta.com/okta_help.htm?id=ext-assign-apps).
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/main/index.md:17]:
```suggestion
* [OpenID Connect client application](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-oidc) in Okta with at least [one user assigned to it](https://help.okta.com/okta_help.htm?id=ext-assign-apps)
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/main/index.md [hunk 4]
```diff
@@ -3,22 +3,26 @@ title: Customize tokens returned from Okta with custom claims
...
+* A [Group in Okta](https://help.okta.com/okta_help.htm?id=ext_Directory_Groups) with at least one person assigned to it.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/main/index.md:18]:
```suggestion
* A [Group in Okta](https://help.okta.com/okta_help.htm?id=ext_Directory_Groups) with at least one person assigned to it
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md [hunk 5]
```diff
@@ -6,24 +6,24 @@ layout: Guides
...
+* Use a static allow list with an authorization server.
 
 **What you need**
 
-* An Okta Developer Edition organization. [Create an org for free](https://developer.okta.com/signup).
-* An [OpenID Connect client application](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-oidc) in Okta with at least [one user assigned to it](https://help.okta.com/okta_help.htm?id=ext-assign-apps).
-* A [group in Okta](https://help.okta.com/okta_help.htm?id=ext_Directory_Groups) with at least one person assigned to it.
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* [OpenID Connect client application](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-oidc) in Okta with at least [one user assigned to it](https://help.okta.com/okta_help.htm?id=ext-assign-apps).
+* A [Group in Okta](https://help.okta.com/okta_help.htm?id=ext_Directory_Groups) with at least one person assigned to it.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md:22]:
```suggestion
* A [Group in Okta](https://help.okta.com/okta_help.htm?id=ext_Directory_Groups) with at least one person assigned to it
```

File: packages/@okta/vuepress-site/docs/guides/validate-id-tokens/main/index.md [hunk 6]
```diff
@@ -32,10 +35,10 @@ Access tokens, on the other hand, are not intended to carry information about th
 
 The high-level overview of validating an ID token looks like this:
 
-- Retrieve and parse your Okta JSON Web Keys (JWK), which should be checked periodically and cached by your application.
-- Decode the ID token, which is in JSON Web Token format
-- Verify the signature used to sign the ID token
-- Verify the claims found inside the ID token
+* Retrieve and parse your Okta JSON Web Keys (JWK), which should be checked periodically and cached by your application.
+* Decode the ID token, which is in JWK format
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/validate-id-tokens/main/index.md:39]:
```suggestion
* Decode the ID token, which is in JWK format.
```

File: packages/@okta/vuepress-site/docs/guides/validate-id-tokens/main/index.md [hunk 7]
```diff
@@ -32,10 +35,10 @@ Access tokens, on the other hand, are not intended to carry information about th
...
+* Verify the signature used to sign the ID token
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/validate-id-tokens/main/index.md:40]:
```suggestion
* Verify the signature used to sign the ID token.
```

File: packages/@okta/vuepress-site/docs/guides/validate-id-tokens/main/index.md [hunk 8]
```diff
@@ -32,10 +35,10 @@ Access tokens, on the other hand, are not intended to carry information about th
...
+* Verify the claims found inside the ID token
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/validate-id-tokens/main/index.md:41]:
```suggestion
* Verify the claims found inside the ID token.
```

**Review by brentschaus-okta — APPROVED** [review]
LGTM

Just a few nits.

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md [hunk 9]
```diff
@@ -6,24 +6,24 @@ layout: Guides
...
+* Use a dynamic allow list with an authorization server.
 
 **What you need**
 
-* An Okta Developer Edition organization. [Create an org for free](https://developer.okta.com/signup).
-* An [OpenID Connect client application](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-oidc) in Okta with at least [one user assigned to it](https://help.okta.com/okta_help.htm?id=ext-assign-apps).
-* A [group in Okta](https://help.okta.com/okta_help.htm?id=ext_Directory_Groups) with at least one person assigned to it.
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* [OpenID Connect client application](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-oidc) in Okta with at least [one user assigned to it](https://help.okta.com/okta_help.htm?id=ext-assign-apps)
+* A [Group in Okta](https://help.okta.com/okta_help.htm?id=ext_Directory_Groups) with at least one person assigned to it.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md]:
```suggestion
* A [Group in Okta](https://help.okta.com/okta_help.htm?id=ext_Directory_Groups) with at least one person assigned to it
```

**Pull request merged**